yt_dlp/extractor/common.py

```diff
@@ -2629,6 +2636,16 @@ def extract_Initialization(source):
                                 'manifest_url': mpd_url,
                                 'filesize': filesize,
                             }
+                        elif mime_type == 'image/jpeg':
+                            f = {
+                                'format_id': format_id,
+                                'ext': 'mhtml',
+                                'manifest_url': mpd_url,
+                                'format_note': 'DASH thumbnail',
```

Comment:
```suggestion
                                'format_note': 'DASH images',
```
is more accurate, no?

Reply:
Technically yes, in practice I have not seen such images contain anything else but grids of thumbnails for preview during seeking.

Reply:
> but grids of thumbnails for preview during seeking.

Interesting, so basically the same thing as https://github.com/ytdl-org/youtube-dl/pull/14951 ?

Can you give me an example URL for this? (also add a test if possible)

Reply:
It seems. <https://github.com/ytdl-org/youtube-dl/issues?q=%22Unknown+MIME+type+image%2Fjpeg+in+DASH+manifest%22> can help searching for examples. I mostly tried it on <https://www.viki.com/videos/1175236v-choosing-spouse-by-lottery-episode-1>, but it has DRM’d main video stream.

Reply:
It doesn't have DRM. Since you said it does, I even actually downloaded and tested...

```
$ yt-dlp https://www.viki.com/videos/1175236v-choosing-spouse-by-lottery-episode-1 -F
[viki] 1175236v: Downloading video JSON
[viki] 1175236v: Downloading MPD manifest
[info] Available formats for 1175236v:
ID             EXT   RESOLUTION FPS |   TBR PROTO | VCODEC        VBR ACODEC     ABR  ASR    NOTE
-------------- ----- ---------- --- - ----- ----- - ----------- ----- --------- ---- ------- ------------------------------------
mpd-http-6     m4a   audio only     |   49k https |                   mp4a.40.2  49k 48000Hz [eng], DASH audio, m4a_dash, 48000Hz
mpd-http-7     m4a   audio only     |   65k https |                   mp4a.40.2  65k 48000Hz [eng], DASH audio, m4a_dash, 48000Hz
mpd-http-5     m4a   audio only     |   81k https |                   mp4a.40.2  81k 48000Hz [eng], DASH audio, m4a_dash, 48000Hz
mpd-http-8     m4a   audio only     |  193k https |                   mp4a.40.2 193k 48000Hz [eng], DASH audio, m4a_dash, 48000Hz
mpd-http-image mhtml unknown        |       mhtml | jpeg                                     DASH thumbnail
mpd-http-1     mp4   426x238    30  |  138k https | avc1.640015  138k                        DASH video, mp4_dash
mpd-http-2     mp4   640x360    30  |  265k https | avc1.64001E  265k                        DASH video, mp4_dash
mpd-http-3     mp4   852x480    30  |  409k https | avc1.64001F  409k                        DASH video, mp4_dash
```